mnd/mnd/management/commands/init.py

```diff
@@ -0,0 +1,18 @@
+from rdrf.management.commands.init import Command as RdrfCommand
+
+from ... import initial_data
+
+
+class Command(RdrfCommand):
```

Comment:
Is it really required to duplicate the management command?
Can't we just use the on in rdrf?

Reply:
No because the version from rdrf also does:

```
from ... import initial_data
```
which loads the initial_data from rdrf and not mnd.